Frends.FixedWidthFlatFile.ConvertToJSON/Frends.FixedWidthFlatFile.ConvertToJSON/FixedWidthFlatFile.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Frends.FixedWidthFlatFile.ConvertToJSON.Definitions;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+
+#pragma warning disable 1591
+
+namespace Frends.FixedWidthFlatFile.ConvertToJSON
+{
+    public static class FixedWidthFlatFile
+    {
+        /// <summary>
+        /// This is a task that converts given value to json string.
+        /// Documentation: https://github.com/CommunityHiQ/Frends.FixedWidthFlatFile.ConvertToJSON
+        /// </summary>
+        /// <param name="input">What value to convert.</param>
+        /// <returns>{string Data} </returns>
+        public static Result ParseJSON(Input data)
```

Comment:
Please add `[PropertyTab]` in the parameter to make the parameter editor seem consistent with other tasks even though there are just one parameter for the task method. Also, since the task contains nested foreach-iterations, it's a good idea to pass cancellation token from Frends to allow cancelling of process execution. So you can add `CancellationToken cancellationToken` after the data parameter. Method name should also be ConvertToJSON instead of ParseJSON.